src/lib/hooks.js

```diff
@@ -1,5 +1,6 @@
 import Triggers from './triggers';
 import CustomFields from './customFields';
+import Language from './language';
```

Comment:
You don't need a `class` to call just one method, you can create a `file` and export the method there.